nexus/flow-rs/src/grpc.rs

```diff
@@ -119,6 +119,8 @@ impl FlowGrpcClient {
                 destination_table_identifier: mapping.destination_table_identifier.clone(),
                 partition_key: mapping.partition_key.clone().unwrap_or_default(),
                 exclude: mapping.exclude.clone(),
+                columns: Default::default(),
+                engine: Default::default(),
```

Comment:
this should be `1` after proto changes are made

Reply:
no, it would be unknown like existing mirrors